### DISCUSSION TITLE
Login in release mode (react-native) fails in the first try

### DISCUSSION THREAD

**Discussion by amine-ba** [Web3Auth/Web3Auth#651] [Help]
Hello im working on a react-native project

the login flow works with no problem on debug mode but always fails in the first attempt on production then works 

here is a snippet of the code

`const web3auth = new Web3Auth(WebBrowser, {
    clientId: clientIdWeb3Auth,
    network: OPENLOGIN_NETWORK.TESTNET,
  });
  const state = await web3auth.login({
    loginProvider: LOGIN_PROVIDER.GOOGLE,
    redirectUrl: redirectUrl
  });`

**Comment by shahbaz17** [comment]
Hey @amine-ba 

Please update to the latest version of [React Native SDK](https://github.com/Web3Auth/web3auth-react-native-sdk) `v3.1.0` and reach out to us if this error is still there.
